apps/journeys-admin/src/components/JourneyList/journeyListData.ts

```diff
@@ -1,13 +1,54 @@
 import { GetJourneys_journeys as Journey } from '../../../__generated__/GetJourneys'
-import { ThemeName, ThemeMode } from '../../../__generated__/globalTypes'
+import {
+  ThemeName,
+  ThemeMode,
+  JourneyStatus
+} from '../../../__generated__/globalTypes'
 
 export const defaultJourney: Journey = {
   __typename: 'Journey',
   id: 'journey-id',
-  title: 'Journey Heading',
-  description: 'Description',
+  title: 'Default Journey Heading',
+  description: null,
   themeName: ThemeName.base,
   themeMode: ThemeMode.light,
   slug: 'default',
-  locale: 'en_US'
+  locale: 'en_US',
+  createdAt: new Date('2021-11-19T12:34:56.647Z'),
+  publishedAt: null,
+  status: JourneyStatus.draft
+}
+
+export const oldJourney: Journey = {
+  ...defaultJourney,
+  title: 'Old Journey Heading',
+  description:
+    'Journey created before the current year should also show the year in the date',
+  createdAt: new Date('2020-11-19T12:34:56.647Z'),
+  publishedAt: new Date('2020-12-19T12:34:56.647Z'),
+  status: JourneyStatus.published
+}
+
+export const publishedJourney: Journey = {
+  ...defaultJourney,
+  title: 'Published Journey Heading',
+  description: ' a published journey',
+  publishedAt: new Date('2021-12-19T12:34:56.647Z'),
+  status: JourneyStatus.published
+}
+
+export const descriptiveJourney: Journey = {
+  __typename: 'Journey',
```

Comment:
```suggestion
...defaultJourney,
```
And remove rest of duplicated values? id, theme, publishedAt etc